blocks/api/source.js

```diff
@@ -1,38 +1,66 @@
-/**
- * WordPress dependencies
- */
-import { createElement } from '@wordpress/element';
+function warnAboutDeprecatedMatcher() {
+	// eslint-disable-next-line no-console
+	console.warn(
+		'Attributes matchers are deprecated and they will be remove in future version of Gutenberg.' +
```

Comment:
Typo: "remove" -> "removed"
Grammar: "in future version" -> "in a future version"